common_types.go

```diff
@@ -153,6 +153,9 @@ type ResourceCreateOptions struct {
 	// in the Application. e.g., when creating API, CanaryRelease, the
 	// Application.ID should be specified.
 	Application *Application
+	// Certificate indicates which Certificate should this resource belong.
+	// This field should be specified when users want to update sub-resources
+	Certificate *Certificate
```

Comment:
Why add the `Certificate` in these options?

Where do you use it?

Reply:
> Why add the `Certificate` in these options?
> 
> Where do you use it?

This field has been removed, I misunderstood it before.